### PR TITLE
Choose supplier feature

### PR DESCRIPTION
**Pull request by ramirez45**
choose seller feature backend

### CODE REVIEW

File: src/main/java/com/revature/configs/SecurityConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+                            response.sendError(
+                                    HttpServletResponse.SC_UNAUTHORIZED,
+                                    ex.getMessage()
+                            );
+                        }
+                )
+                .and();
+
+        // Set permissions on endpoints
+//        http.authorizeRequests()
```
Comment (iatenine) [inline src/main/java/com/revature/configs/SecurityConfiguration.java]: Why is this being removed? Is it established elsewhere?

File: src/main/java/com/revature/controllers/SellerController.java [hunk 2]
```diff
@@ -0,0 +1,37 @@
...
+import java.util.Optional;
+
+@CrossOrigin
+@RestController
+public class SellerController {
+
+    @Autowired
+    SellerService sellerService;
+
+//    @GetMapping("/shop_products/{id}")
```
Comment (iatenine) [inline src/main/java/com/revature/controllers/SellerController.java]: There are no routes available in this function. Should the file be removed completely?

**Review by iatenine — CHANGES_REQUESTED** [review]
Need some changes as specified in review + merge conflicts resolved between this branch and `dev` before merge

File: src/main/java/com/revature/models/ShopProduct.java [hunk 3]
```diff
@@ -20,7 +20,7 @@
     @Id
     @GeneratedValue(strategy = GenerationType.IDENTITY)
     @Column(name = "shop_product_id")
-    private int id;
+    private int shop_product_id;
```
Comment (RichardSimpson235) [inline src/main/java/com/revature/models/ShopProduct.java]: Why was this changed?

**Comment by iatenine** [comment]
Seems to be redundant due to a previous merge

**Pull request closed without merging**